projects/packages/scheduled-updates/src/class-scheduled-updates.php

```diff
@@ -226,6 +235,17 @@ function ( $event ) use ( $updated_at ) {
 	 */
 	public static function clear_cron_cache() {
 		wp_cache_delete( 'alloptions', 'options' );
+		wp_cache_delete( 'notoptions', 'options' );
```

Comment:
Should this function be named `clear_options_cache`?

Can you tell me more about clearing notoptions? I'd expect `cron` to never be part of that, is that not the case?

Reply:
> Deleting just the cron entry does not solve the problems.

Do you know why that is?

Reply:
We had a very long debug session but we cannot find the exact reason why. Probably something related to other plugins running on WoA (like the `action-scheduler`). p1715925157297969-slack-C01A60HCGUA

Reply:
Can you tell me more about clearing notoptions? I'd expect cron to never be part of that, is that not the case?

Reply:
I think this is probably due to the timing we tried to clear the cache wrong before. We used
```
wp_cache_delete( 'alloptions', 'options' );
wp_cache_delete( 'notoptions', 'options' );
``` 
By looking into the codebase and see what's the proper way of clearing all the cache, so place it for testing, but I agree if we can only aim for the `cron` option that'll be the best. In our current implementation, I tried to test it out again with only:

```
wp_cache_delete( 'cron', 'options' );
wp_load_alloptions( true );
```

And looks like it works too, @zaerl could you change it back to this and see if it works for you?

Reply:
> And looks like it works too, @zaerl could you change it back to this and see if it works for you?
Confirmed it is working https://github.com/Automattic/jetpack/pull/37399/commits/8ee38f6f60dde28d95d4bf919c211d4249c3fd0d